infra/core/tenant/role-definition/cluster-custom-resources-reader-role.bicep

```diff
@@ -0,0 +1,36 @@
+targetScope = 'subscription'
+
+@description('Name of the Custom Role Definition in Azure.')
+param roleName string
+
+@description('Array of scopes to assign the role to.')
+param roleScopes array
+
+@description('Detailed description of the role definition')
+param roleDescription string = 'Custom Role to read cluster data.'
+
+resource asbCustomRoleResource 'Microsoft.Authorization/roleDefinitions@2022-04-01' = {
```

Comment:
why is the resource named as asbCustomRoleResource? could you correct the name please.

Reply:
Fixed